lib/screens/v2/candidate_generator/widgets/departures.ex

```diff
@@ -79,24 +108,80 @@ defmodule Screens.V2.CandidateGenerator.Widgets.Departures do
     [departures_instance]
   end
 
+  # When a section has no departures on a screen with multiple sections, populate it with a "no
+  # data" entry. This may include a "representative" route for the section, used to determine an
+  # icon to display alongside the message.
+  #
+  # NOTE: Assumes any given section is configured such that it only displays departures from a
+  # single route-type-or-subway-line. If there would be more than one route-type-or-subway-line,
+  # one is picked arbitrarily.
+  defp post_process_no_data({:ok, []}, section, true = _has_multiple_sections, route_fetch_fn) do
+    %Section{
+      query: %Query{
+        params: %Query.Params{route_ids: route_ids, route_type: route_type, stop_ids: stop_ids}
+      }
+    } = section
+
+    fetch_params =
+      Map.reject(
+        %{
+          limit: 1,
+          ids: route_ids,
+          route_types: if(route_type, do: [route_type], else: []),
+          stop_ids: stop_ids
+        },
+        fn {_key, value} -> value == [] end
+      )
+
+    route =
+      case route_fetch_fn.(fetch_params) do
+        {:ok, [route]} -> route
+        :error -> nil
+      end
```

Comment:
This is raising a `CaseClauseError` in dev. Looks like we just need to adjust the pattern matches.

[Sentry - SCREENS-A5](https://mbtace.sentry.io/issues/5459051453/)